hbase_sql/pom.xml

```diff
@@ -90,7 +90,7 @@
         <dependency>
             <groupId>com.typesafe.akka</groupId>
             <artifactId>akka-remote_${scala.binary.version}</artifactId>
-            <version>2.4.8</version>
+            <version>${akka.version}</version>
```

Comment:
This is an unwanted dependency. We provide it as part of Splice. Do we really need to upgrade it for Spark 3.0? (I'd rather eliminate it)

Reply:
It's used in hbase_sql/com.splicemachine.olap.OlapJobStatus

`import akka.remote.FailureDetector;
import akka.remote.FailureDetector$;
import akka.remote.PhiAccrualFailureDetector;`

should it be refactored somehow?

Reply:
We should not bring a dependency on akka just because we want to use PhiAccrualFailureDetector.
Refactoring it goes beyond the scope of this JIRA.
Elevating this dependency to the parent pom.xml creates a perception of its legitimacy. I would leave it confined to hbase_sql/pom.xml as is (no version upgrade to avoid potential unwanted transitive dependencies).

Reply:
I should upgrade the version, as there is no 2.4.8 with scala 2.12. The closest version with 2.12 support was 2.4.12. I can directly update the version in hbase_sql/pom.xml, without the parent change.

Reply:
That's the way to do it for now (hopefully, 2.4.12 does not bring new transitive dependencies)

Reply:
Done